tests/list

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+find './ditdir' -type f | sort
```

Comment:
It's better to repeat the code of `step` here, so in the `.ok` file we see what's being done exactly.